website/content/en/pre-docs/concepts/_index.md

```diff
@@ -0,0 +1,167 @@
+
+* **Provisioner CR**: Karpenter defines a Custom Resource called a Provisioner to specify provisioning configuration.
+Each provisioner manages a distinct set of nodes, but pods can be scheduled to any provisioner that supports its scheduling constraints.
+A provisioner contains constraints that impact the nodes that can be provisioned and attributes of those nodes (such timers for removing nodes).
+See [Provisioner](/docs/provisioner-crd/) for a description of settings and the [Provisioning](/docs/tasks/provisioner.md) task for of provisioner examples. 
+
+* **Well-known labels**: The provisioner can use well-known Kubernetes labels to allow pods to request only certain instance types, architectures, operating systems, or other attributes when creating nodes.
+See [Well-Known Labels, Annotations and Taints](https://kubernetes.io/docs/reference/labels-annotations-taints/) for details.
+Keep in mind that only a subset of these labels are supported in Karpenter, as described later.
+
+* **Deprovisioning nodes**: A provisioner can also include time-to-live values to indicate when nodes should be deprovisioned after a set amount of time from when they were created or after they becomes empty of deployed pods.
+
+* **Multiple provisioners**: Multiple provisioners can be configured on the same cluster.
+For example, you might want to configure different teams on the same cluster to run on completely separate capacity.
+One team could run on nodes nodes using BottleRocket, while another uses EKSOptimizedAMI.
+
+Although most use cases are addressed with a single provisioner for multiple teams, multiple provisioners are useful to isolate nodes for billing, use different node constraints (such as no GPUs for a team), or use different deprovisioning settings.
+
+### Deprovisioning nodes
+
+Karpenter deletes nodes when they are no longer needed.
+
+* **Finalizer**: Karpenter places a finalizer bit on each node it creates.
+When a request comes in to delete one of those nodes (such as a TTL or a manual `kubectl delete node`), Karpenter will cordon the node, drain all the pods, terminate the EC2 instance, and delete the node object.
+Karpenter handles all clean-up work needed to properly delete the node.
+* **Node Expiry**: If a node expiry time-to-live value (`ttlSecondsUntilExpired`) is reached, that node is drained of pods and deleted (even if it is still running workloads).
+* **Empty nodes**: When the last workload pod running on a Karpenter-managed node is gone, the node is annotated with an emptiness timestamp.
+Once that "node empty" time-to-live (`ttlSecondsAfterEmpty`) is reached, finalization is triggered.
+
+For more details on how Karpenter deletes nodes, see [Deleting nodes with Karpenter](/docs/tasks/delete-nodes.md) for details.
+
+### Upgrading nodes
+
+A straight-forward way to upgrade nodes is to set `ttlSecondsUntilExpired`.
+Nodes will be terminated after a set period of time and will be replaced with newer nodes.
```

Comment:
Are there any safeguards against multiple nodes being terminated at once? If all my nodes are created at the same time will they all be destroyed at the same time too? Do we have a way to add jitter or skew to this TTL or locking to make sure the entire cluster doesn't recycle at once?